test/spinxsk/spinxsk.c

```diff
@@ -920,6 +939,10 @@ CleanupQueue(
         ASSERT_FRE(res);
     }
 
+    if (Queue->fnmp != NULL) {
+	    FnMpClose(Queue->fnmp);
```

Comment:
this alignment looks fishy - are there hard tabs (not spaces) here?

Reply:
Good catch. I am not sure what happened, my editor got very confused with tab and spaces on this project and I must have missed that one when fixing it.